doc/rainbow-delimiters.txt

```diff
@@ -145,9 +145,9 @@ Alternatively, the same configuration in Lua:
                                                     *rainbow-delimiters.setup*
 'rainbow-delimiters.setup'.setup({config})
 
-Some people prefer to call a Lua `setup` function, so a setup function is
-available as part of a Lua module.
->
+While |g:rainbow_delimiters| is the recommended configuration method, a Lua
+`setup` function is available for compatibility with common plugin managers.
+>lua
     require 'rainbow-delimiters.setup'.setup {
```

Comment:
I think there was a metatable update recently that simplified this, so you can just use `require 'rainbow-delimiters'.setup` (which will also work with e.g. Lazy). @HiPhish will know if this is true. 

If so, this could be simplified using that fact.